docs/source/mps.rst

```diff
@@ -0,0 +1,10 @@
+torch.backends.mps
```

Comment:
I already included that in one of my commits for the previous PR: https://github.com/pytorch/pytorch/blame/master/docs/source/backends.rst#L81-L87

This file can just be deleted.

Reply:
Done.